### PR TITLE
Add Contracts for Cairo docs

### PR DESCRIPTION
**Pull request by martriay**
Depends on https://github.com/OpenZeppelin/cairo-contracts/pull/396.

### CODE REVIEW

File: ui/theme/partials/navigation.hbs [hunk 1]
```diff
@@ -64,12 +65,12 @@
...
+      {{> navigation-tree items=@root.page.navigation}}
     </ol>
     {{/if}}
   </div>
 
   {{/with}}
   {{/if}}
 
-</nav>
+</nav>
```
Comment (martriay) [inline ui/theme/partials/navigation.hbs]:
```suggestion
</nav>

```

File: ui/theme/partials/navigation.hbs [hunk 2]
```diff
@@ -42,18 +43,18 @@
...
-                {{#unless (eq this ../latest)}}
-                  <li>
-                    <a href="{{{url}}}">{{displayVersion}}</a>
-                  </li>
-                {{/unless}}
+              {{#unless (eq this ../latest)}}
+              <li>
+                <a href="{{{url}}}">{{displayVersion}}</a>
+              </li>
+              {{/unless}}
```
Comment (martriay) [inline ui/theme/partials/navigation.hbs]:
```suggestion
                {{#unless (eq this ../latest)}}
                  <li>
                      <a href="{{{url}}}">{{displayVersion}}</a>
                  </li>
                {{/unless}}
```

File: ui/theme/partials/navigation.hbs [hunk 3]
```diff
@@ -42,18 +43,18 @@
             <ul>
               <li>
                 {{#with latest}}
-                  <a href="{{{url}}}">{{displayVersion}}</a>
+                <a href="{{{url}}}">{{displayVersion}}</a>
```
Comment (martriay) [inline ui/theme/partials/navigation.hbs]:
```suggestion
                  <a href="{{{url}}}">{{displayVersion}}</a>
```

File: ui/theme/partials/navigation.hbs [hunk 4]
```diff
@@ -2,25 +2,26 @@
...
+        d="M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z" />
     </svg>
   </button>
 
   {{#if (eq @root.page.component.name "ROOT")}}
   <div class="nav-components">
     <h3 class="nav-title">Start here</h3>
     {{#each (pick site.components "ROOT, learn")}}
-      {{> navigation-component }}
+    {{> navigation-component }}
```
Comment (martriay) [inline ui/theme/partials/navigation.hbs]:
```suggestion
      {{> navigation-component }}
```

File: ui/theme/partials/navigation.hbs [hunk 5]
```diff
@@ -2,25 +2,26 @@
...
     {{/each}}
 
     <h3 class="nav-title">Our Products</h3>
-    {{#each (pick site.components "defender, contracts")}}
-      {{> navigation-component }}
+    {{#each (pick site.components "defender, contracts, contracts-cairo")}}
+    {{> navigation-component }}
```
Comment (martriay) [inline ui/theme/partials/navigation.hbs]:
```suggestion
      {{> navigation-component }}
```

File: ui/theme/partials/navigation.hbs [hunk 6]
```diff
@@ -2,25 +2,26 @@
...
     {{/each}}
 
     <h3 class="nav-title">More</h3>
     {{#each (pick site.components "upgrades-plugins, subgraphs") }}
-      {{> navigation-component }}
+    {{> navigation-component }}
```
Comment (martriay) [inline ui/theme/partials/navigation.hbs]:
```suggestion
      {{> navigation-component }}
```

File: ui/theme/partials/navigation.hbs [hunk 7]
```diff
@@ -64,12 +65,12 @@
     </div>
     {{#if (eq this @root.page.component)}}
     <ol class="nav-list">
-    {{> navigation-tree items=@root.page.navigation}}
+      {{> navigation-tree items=@root.page.navigation}}
```
Comment (martriay) [inline ui/theme/partials/navigation.hbs:68]:
```suggestion
        {{> navigation-tree items=@root.page.navigation}}
```

**Review by martriay — COMMENTED** [review]
oops, formatting mistakes

**Review by frangio — APPROVED** [review]

**Pull request merged**